ztp/siteconfig-generator/siteConfig/testdata/SplitAndMergeMachineConfigExpected/mergeMC.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
         agent-install.openshift.io/install-config-overrides: '{"networking":{"networkType":"OVNKubernetes"}}'
         argocd.argoproj.io/sync-wave: "1"
         ran.openshift.io/ztp-gitops-generated: '{}'
+        ran.openshift.io/ztp-warning/field-deprecation/cpuset: siteConfig.clusters[].node[].cpuset will be deprecated after OCP 4.15, please use siteConfig.clusters[].cpuPartitioningMode for OCP versions >= 4.13
```

Comment:
Yeah, since we won't know the version the customer chooses for openshift, figured if they include a `cpuset` we notify them that for 4.13 onwards they need to switch over to the new method.